library/ssl_tls.c

```diff
@@ -4158,6 +4219,10 @@ static const unsigned char ssl_serialized_session_header[] = {
  * #if defined(MBEDTLS_HAVE_TIME)
  *         case server: uint64 ticket_creation_time;
  * #endif
+ * #if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_ALPN)
+ *                      uint8 alpn_len;
+ *                      opaque ticket_alpn<0..255>;
```

Comment:
```suggestion
 *                      opaque ticket_alpn<0..255>;
```
This alone defines that there is one byte of length before the `ticket_alpn` value.